java/core/src/main/java/com/whylogs/core/metrics/IntegralMetric.java

```diff
@@ -0,0 +1,99 @@
+package com.whylogs.core.metrics;
+
+import com.whylogs.core.PreProcessedColumn;
+import com.whylogs.core.SummaryConfig;
+import com.whylogs.core.metrics.components.MaxIntegralComponent;
+import com.whylogs.core.metrics.components.MinIntegralComponent;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+
+@Getter
+@EqualsAndHashCode(callSuper=false)
+public class IntegralMetric extends BaseMetric<IntegralMetric> {
+    public static final String DEFAULT_NAMESPACE = "ints";
```

Comment:
Maybe just `NAMESPACE`? "Default" suggests it can be changed.

Reply:
Ahh I went for default since you can pass in a different namespace, but I like the change to just namespace.

Reply:
Why are we allowing it to change namespaces? That seems dangerous :)

Reply:
I agree depending on how we do namespaces. :) I thought I saw it had a change constructor in python but just double checking it didn't. :P So I'll change that. 

My thought had been if you're doing a namespace to do like metrics but wanted and integral metric that wasn't mergable with another IntegralMetric for some reason. Having the ability to change the namespace might allow that, which I thought might be helpful with custom. That being said that would add a whole list of interesting cases so not allowing that would be simpler and safer.

Reply:
(also it didn't allow it to change after init, just give a different namespace passed in)

Reply:
You can always make a subclass if you want to change the namespace. I think that emphasizes that you're making a completely different metric if you change the namespace.

Reply:
Totally. This change is already in